content/shared/influxctl/table/create.md

```diff
@@ -0,0 +1,101 @@
+> time format to use in your partition template.
+> Otherwise, InfluxDB omits time from the partition template and won't compact partitions.
+
+## Usage
+
+```sh
+influxctl table create [flags] <DATABASE_NAME> <TABLE_NAME>
+```
+
+## Arguments
+
+| Argument          | Description                 |
+| :---------------- | :-------------------------- |
+| **DATABASE_NAME** | Name of the target database |
+| **TABLE_NAME**    | Table name                  |
+
+## Flags
+
+| Flag |                         | Description                                                                                                                              |
+| :--- | :---------------------- | :--------------------------------------------------------------------------------------------------------------------------------------- |
+|      | `--template-tag`        | Tag to add to partition template (can include multiple of this flag)                                                                     |
+|      | `--template-tag-bucket` | Tag and number of buckets to partition tag values into separated by a comma--for example: `tag1,100` (can include multiple of this flag) |
+|      | `--template-timeformat` | Timestamp format for partition template <!--(default is `%Y-%m-%d`)-->                                                                   |
+| `-h` | `--help`                | Output command help                                                                                                                      |
+
+{{% caption %}}
+_Also see [`influxctl` global flags](/influxdb3/version/reference/cli/influxctl/#global-flags)._
+{{% /caption %}}
+
+## Examples
+
+- [Create a table](#create-a-table)
+- [Create a table with a custom partition template](#create-a-table-with-a-custom-partition-template)
+
+In the following examples, replace:
+
+- {{% code-placeholder-key %}}`DATABASE_NAME`{{% /code-placeholder-key %}}:
+  The name of the database to create the table in.
+- {{% code-placeholder-key %}}`TABLE_NAME` {{% /code-placeholder-key %}}:
+  The name of table to create.
+
+### Create a table
+
+{{% code-placeholders "(DATABASE|TABLE)_NAME" %}}
+```sh
+influxctl table create DATABASE_NAME TABLE_NAME
+```
+{{% /code-placeholders %}}
+
+### Create a table with a custom partition template
+
+The following example creates a new table and applies a partition
+template that partitions by two tags (`room` and `sensor-type`) and by day using
+the time format `%Y-%m-%d`:
+
+{{% code-placeholders "(DATABASE|TABLE)_NAME" %}}
+```sh
+influxctl table create \
+  --template-tag room \
+  --template-tag sensor-type \
+  --template-tag-bucket customerID,1000 \
+  --template-timeformat '%Y-%m-%d' \
+  DATABASE_NAME \
+  TABLE_NAME
+```
+{{% /code-placeholders %}}
+
+_For more information about custom partitioning, see
+[Manage data partitioning](/influxdb3/version/admin/custom-partitions/)._
+
+{{% expand "View command updates" %}}
```

Comment:
Doesn't need a wrapper? I don't think I've tried without it.

Reply:
No, it doesn't need a wrapper. The wrapper provides applies the bottom border to the last child in the element. Without the wrapper, each expand element only has a top border. Where this is the last block on the page, the hr at the bottom of the page functions as the bottom-border of the expand block.